packages/bpk-component-modal/src/BpkModal.js

```diff
@@ -29,13 +29,17 @@ const ScrimBpkModalDialog = withScrim(BpkModalDialog);
 
 const BpkModal = (props) => {
   const {
-    isOpen, onClose, target, ...rest
+    isOpen,
+    onClose,
+    target,
+    renderTarget,
+    ...rest
   } = props;
 
   delete rest.onClose;
 
   return (
-    <Portal isOpen={isOpen} onClose={onClose} target={target}>
+    <Portal isOpen={isOpen} onClose={onClose} target={target} renderTarget={renderTarget}>
```

Comment:
So to support theming the users has to supply `renderTarget` otherwise it will not work?